lib/grape-swagger/doc_methods.rb

```diff
@@ -491,7 +497,7 @@ def setup(options)
         end
 
         ops = visible_ops.group_by do |route|
-          @@documentation_class.parse_path(route.route_path, api_version)
+          @@documentation_class.parse_path(route.route_path, api_version, hide_module_from_path)
```

Comment:
I initially did it that way, but had some problem I can't remember. When I fix up again, I"ll try with class var and see if tests pass